### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by lahiru823**

### CODE REVIEW

File: modules/jaggery-apps/app-store-web/extensions/assets/webapp/themes/store/js/logic/asset/overview/overview.js [hunk 1]
```diff
@@ -47,4 +47,20 @@ $(function () {
             }
         }
     });
+});
+
+$(document).ready(function () {
```
Comment (lahiru823) [inline modules/jaggery-apps/app-store-web/extensions/assets/webapp/themes/store/js/logic/asset/overview/overview.js]: client side security check was moved to server side check in subsequent PR

File: modules/jaggery-apps/app-publisher-web/themes/appm/js/policy.group.js [hunk 2]
```diff
@@ -221,7 +223,7 @@ function savePolicyGroupData(isSaveAndClose) {
         else {
             updatePolicyGroup(policyGroupName, throttlingTier, anonymousAccessToUrlPattern, userRoles, JSON.stringify(objPartialMappings.policyGroupOptions), isSaveAndClose, policyGroupDesc);
 
-            updatePolicyGroupPartialXACMLPolicies($("#uuid").val());
+           // updatePolicyGroupPartialXACMLPolicies($("#uuid").val());
```
Comment (rushmin) [inline modules/jaggery-apps/app-publisher-web/themes/appm/js/policy.group.js]: Remove commented code.

File: modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag [hunk 3]
```diff
@@ -307,6 +307,13 @@ require('/modules/store.js').exec(function (ctx) {
         skipGateway = false;
     }
 
+
+    var process = require('process');
+    //set local ip
```
Comment (rushmin) [inline modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag]: Comments should start with a capital letter.

File: modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag [hunk 4]
```diff
@@ -307,6 +307,13 @@ require('/modules/store.js').exec(function (ctx) {
...
+    asset.attributes.localIP = "https://" + process.getProperty('server.host');
+    //set http port
```
Comment (rushmin) [inline modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag]: Comments should start with a capital letter.

File: modules/jaggery-apps/app-publisher-web/themes/appm/js/policy.group.js (hunk 2 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-publisher-web/themes/appm/js/policy.group.js]: Already fixed with a subsequent fix

File: modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag (hunk 3 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag]: commit# https://github.com/wso2/carbon-appmgt/commit/1bbd23fdbedf6fd09240cde99b2d470ef33b5fac

File: modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag (hunk 4 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/asset.jag]: commit# https://github.com/wso2/carbon-appmgt/commit/1bbd23fdbedf6fd09240cde99b2d470ef33b5fac

**Pull request merged**